custom/icds_reports/tests/agg_tests/reports/test_awc_reports.py

```diff
@@ -75,6 +75,26 @@ def test_beneficiary_details_have_age_in_month_not_have_recorded_height(self):
         self.assertEqual(data['person_name'], 'Name 3483')
         self.assertEqual(data['mother_name'], 'रींकीकुँवर')
 
+    def test_beneficiary_details_status_active(self):
+        data = get_beneficiary_details(
+            case_id='411c4234-8475-415a-9c28-911b85868aa5',
+            awc_id='a15',
+            selected_month=(2017, 6, 1)
```

Comment:
how are you getting this month? as far as I know, we only aggregate till 2017-05

Reply:
it is there from earlier. So just used it

Reply:
checked it. This is using this filter (month__range=(six_month_before, selected_month))